lib/bundle.js

```diff
@@ -164,7 +165,7 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     if (opts.injectConfig)
       ui.log('ok', '`' + output.bundleName + '` added to config bundles.');
 
-    logBuild(path.relative(process.cwd(), fileName), opts);
+    logBuild(path.relative(env.cwd(), fileName), opts);
```

Comment:
Note though that the `cwd` setting in `jspm.js` should be working here for that anyway I think?